RTNeural/activation/activation_eigen.h

```diff
@@ -94,6 +94,36 @@ class SigmoidActivation : public Activation<T>
     Eigen::Matrix<T, Eigen::Dynamic, 1> outVec;
 };
 
+template <typename T>
+class SoftmaxActivation : public Activation<T>
+{
+public:
+    SoftmaxActivation(size_t size)
+        : Activation<T>(size, {}, "softmax")
+    {
+        inVec = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>::Zero(size, 1);
+        outVec = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>::Zero(size, 1);
+    }
+
+    SoftmaxActivation(std::initializer_list<size_t> sizes)
+        : SoftmaxActivation(*sizes.begin())
+    {
+    }
+
+    inline void forward(const T* input, T* out) override
+    {
+        inVec = Eigen::Map<const Eigen::Matrix<T, Eigen::Dynamic, 1>>(
```

Comment:
After [some commits](https://github.com/jatinchowdhury18/RTNeural/commit/802e85342ae8e01e6f631ed48674929ddda53409#diff-a47ad744e59f0496fa2ccebb9d8a56a36aa4a0747a75966d5c032f2026a9bbdeR27) that I added last weekend, we can actually use `Eigen::Map` with the `Eigen::Aligned16` template argument. You might need to rebase your branch for this to work though.

Reply:
done!